HW1.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+# create a 2pi sequence and plug it in to sine function and cosine function
+x = np.arange(0,2 * np.pi,0.001)
+y_sine = np.sin(x)
+y_cosine = np.cos(x)
+
+# plot them out
+plt.plot(x, y_sine, label = "Sine")
```

Comment:
Unexpected spaces around the function parameters. 
```suggestion
plt.plot(x, y_sine, label="Sine")
```